src/trading/consts.ts

```diff
@@ -1,6 +1,11 @@
 import { EcdsaSigningScheme, SigningScheme } from '../order-book'
 
-export const log = (text: string) => console.log(`[COW TRADING SDK] ${text}`)
+export const log = (text: string) => {
```

Comment:
Its not the original signature of the logs, but enough if we just log a message.

Reply:
why a log is a `const`?

Reply:
<img width="1069" alt="image" src="https://github.com/user-attachments/assets/28ba29c1-b5cb-4814-9a99-8a9cf39e1f1e" />

I don't have a strict preference on that.
We use both styles of function declaration `function` and `const`.
Should I declare that as `function`?

Reply:
I personally prefer `function` unless there's a reason for using `const`, such as applying a type/interface to it.
In my mind: if there's a language keyword for it, use that.

Reply:
I love your explanation Leandro!
Added it to the doc: https://www.notion.so/cownation/Development-Rules-19d58a396d924640bcf6ae1c4c167521?pvs=4#18b8da5f04ca8018ba5fdc8e4917ca76
I hope eventually we will migrate all the rules into eslint.

Reply:
Changed to function btw